packages/babel-core/test/api.js

```diff
@@ -540,7 +540,7 @@ describe("api", function () {
         column: 4,
       }),
     ).toEqual({
-      name: null,
+      name: "Foo",
```

Comment:
The "Foo" name doesn't exist in the input sourcemap, but does in the transformed sourcemap. Remapping will use the input's if it existed, but falls back to the transformed one if not.